configs/azure_c_shared_utilityFunctions.cmake

```diff
@@ -484,6 +484,16 @@ function(set_platform_files c_shared_dir)
         if(${use_condition})
             set(CONDITION_C_FILE ${c_shared_dir}/adapters/condition_win32.c PARENT_SCOPE)
         endif()
+        if(NOT ${use_etw})
+            set(XLOGGING_C_FILE ${c_shared_dir}/src/xlogging.c PARENT_SCOPE)
+            set(LOGGING_C_FILE ${c_shared_dir}/src/consolelogger.c PARENT_SCOPE)
+            set(LOGGING_H_FILE ${c_shared_dir}/inc/azure_c_shared_utility/consolelogger.h PARENT_SCOPE)
```

Comment:
done.

---
In reply to: [126213933](https://github.com/Azure/azure-c-shared-utility/pull/89#discussion_r126213933) [](ancestors = 126213933)